rust/src/ike/logger.rs

```diff
@@ -214,12 +214,14 @@ fn log_ikev2(tx: &IKETransaction, jb: &mut JsonBuilder) -> Result<(), JsonError>
     jb.open_object("ikev2")?;
 
     jb.set_uint("errors", tx.errors as u64)?;
-    jb.open_array("notify")?;
-    for notify in tx.notify_types.iter() {
-        jb.append_string(&format!("{:?}", notify))?;
+    if tx.notify_types.len() > 0 {
```

Comment:
Our friend clippy will probably complain here and suggest:
```
if !ts.notify_types.is_empty() {
```

Reply:
Replacing this.

By the way `git grep "len() > 0" rust/ | wc -l` has 123 occurrences...